docker-for-windows/faqs.md

```diff
@@ -68,29 +56,29 @@ disk:
     want to keep. (See [export](/engine/reference/commandline/export) in the
     Docker Engine command line reference.)
 
-3.  Uninstall the current app & Install a different version of the app (Stable
+3.  Uninstall the current app and install a different version of the app (Stable
     or Edge), or reset your VM disk.
 
 5.  Use `docker load -i images.tar` to reload previously saved images. (See
-    [load](/engine/reference/commandline/load) in the Docker Engine
+    [load](/engine/reference/commandline/load) in the Docker Engine.
 
 6.  Use `docker import -i myContainer1.tar` to create a filesystem image
     corresponding to previously exported containers. (See
-    [import](/engine/reference/commandline/import) in the Docker Engine
+    [import](/engine/reference/commandline/import) in the Docker Engine.
 
 [This
 procedure](https://docs.docker.com/storage/volumes/#backup-restore-or-migrate-data-volumes)
 explains how to backup and restore data volumes.
 
 ## Experimental features
 
-{% include experimental-feature.md %}
+{% include experimental.md %}
 
 ## Feedback
 
 ### What kind of feedback are we looking for?
 
-Everything is fair game. We'd like your impressions on the download-install
+Everything is a fair game. We'd like your impressions on the download-install
```

Comment:
This is an incorrect correction: the previous version ("Everything is fair game") is the normal expression.

Reply:
Fixed in the review commit.